src/v3/test/integration/authenticator-enroll-security-question.test.tsx

```diff
@@ -103,6 +106,266 @@ describe('authenticator-enroll-security-question', () => {
         expect(answerFieldError).toBeNull();
       });
     });
+
+    it('should show field level character count error message when invalid number of characters are sent and field should retain characters', async () => {
+      const {
+        user, authClient, findByText, findByTestId,
+      } = await setup({ mockResponse: predefinedQuestionAnswerErrorResponse });
+
+      await findByText(/Set up security question/);
+      const submitButton = await findByText('Verify', { selector: 'button' });
+      const answerEle = await findByTestId('credentials.answer') as HTMLInputElement;
+
+      const answer = 'pi';
+      await user.type(answerEle, answer);
+      expect(answerEle.value).toEqual(answer);
+
+      await user.click(submitButton);
+      expect(authClient.options.httpRequestClient).toHaveBeenCalledWith(
+        'POST',
+        'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+        {
+          data: JSON.stringify({
+            credentials: {
+              questionKey: 'disliked_food',
+              answer,
+            },
+            stateHandle: 'fake-stateHandle',
+          }),
+          headers: {
+            Accept: 'application/json; okta-version=1.0.0',
+            'Content-Type': 'application/json',
+            'X-Okta-User-Agent-Extended': 'okta-auth-js/9.9.9',
```

Comment:
what's this 9.9.9?

Reply:
This was from the newly created mock file, I will change this to match the other mock files.

Reply:
Actually, sorry, this value is used in every integration test, it is set this way when the mock authClient is created. You can see it is explicitly set here: https://github.com/okta/okta-signin-widget/blob/0.0/src/v3/test/integration/util/createAuthClient.ts#L153-L156